components/BottomNavigation/tests/unit/BottomNavigationItemViewTests.m

```diff
@@ -127,6 +127,27 @@ - (void)testSetSelectedItemTintColorUpdatesInkColor {
   XCTAssertNotEqualObjects(item2.inkView.inkColor, item2DefaultInkColor);
 }
 
+- (void)testSetTitleVisibilityUpdatesLayout {
+  // Given
+  MDCBottomNavigationItemView *view = [[MDCBottomNavigationItemView alloc] init];
+  view.title = @"Test Content";
+  view.image = fakeImage();
+  view.bounds = CGRectMake(0, 0, 100, 100);
+  view.contentVerticalMargin = 20;
+  view.contentHorizontalMargin = 20;
+  view.titleVisibility = MDCBottomNavigationBarTitleVisibilityAlways;
+  [view layoutIfNeeded];
+  CGFloat imageMidYWithTitle = CGRectGetMidY(view.iconImageView.frame);
+
+  // When
+  view.titleVisibility = MDCBottomNavigationBarTitleVisibilityNever;
+  [view layoutIfNeeded];
+
+  // Then
+  CGFloat imageMidYWithoutTitle = CGRectGetMidY(view.iconImageView.frame);
+  XCTAssertGreaterThan(imageMidYWithoutTitle, imageMidYWithTitle + 1);  // We add 1 to make sure the change in position is greater than a rounding error. Practically, it's likely to be greater than 10.
```

Comment:
Nit: This line looks a bit long. Consider breaking it up or running clang-format on your changes.

Reply:
Broken up and clang-formatted.